### ISSUE TITLE
[Feature request] Change `makeknownhosts` options to match other `make*` commands

### ISSUE THREAD

**Issue by samveen** [xcat2/xcat-core#2939]
As of now, most `make*` commands use the option `-d` to `delete`/`remove` configuration for noderanges.

However, the `makeknownhosts` command doesn't follow that logic. Please add `-d` as an alias for `-r` for the `makeknownhosts` command:

`make*` commands:

```
[samveen@xcatmn1 upgrade]$ man makehosts |grep -- -d
       makehosts [-n] [noderange] [-l|--longnamefirst] [-d] [-m|--mactolinklocal]
       -d    Delete rather than create records. This will also delete any additional network interfaces (NICs) included in the node definitions.
[samveen@xcatmn1 upgrade]$ man makedns |grep -- -d
       makedns [-V|--verbose] [-e|--external] [-d|--delete noderange]
       -d|--delete
          makedns -d node1
[samveen@xcatmn1 upgrade]$ man makedhcp |grep -- -d
       makedhcp -a -d [-l|--localonly]
       makedhcp -d noderange [-l|--localonly]
       If you need to delete node entries from the DHCP configuration, use the -d flag.
       -d noderange
       -a -d       Delete all node entries, that were added by xCAT, from the DHCP server configuration.
[samveen@xcatmn1 upgrade]$ man makeconservercf |grep -- -d
       makeconservercf [-V|--verbose] [-d|--delete] [noderange]
       If -d is specified, makeconservercf will remove specified nodes from /etc/conserver.cf file. If noderange is not specified, all xCAT nodes
       -d|--delete
          makeconservercf -d node01
[samveen@xcatmn1 upgrade]$ man makeroutes |grep -- -d
       makeroutes [-d|--delete] [-r|--routenames r1[,r2...]]
       makeroutes noderange [-d|--delete] [-r|--routenames r1[,r2...]]
       -d|--delete
          makeroutes node1,node2 -d -r rr1,rr2
```
`makeknownhosts`:
```
[samveen@xcatmn1 upgrade]$ man makeknownhosts |grep -- -d
[samveen@xcatmn1 upgrade]$ man makeknownhosts |grep -- -r
       makeknownhosts noderange [-r|--remove] [-V|--verbose]
       -r|--remove
          makeknownhosts node02 -r
```

**Comment by samveen** [comment]
@immarvin, @cxhong  If you want, I can submit a MR for this, as it is a trivial change.

**Comment by immarvin** [comment]
hi @samveen , sure, it will be appreciated

thanks

**Comment by cxhong** [comment]
@samveen,  let me know if you will submit a PR for this. otherwise, I will try to fix it.  Thanks.

**Comment by samveen** [comment]
I'll get this done, however I'll need a couple of days.

**Comment by zet809** [comment]
Seems there is issue for the fix, so reopen this problem.

**Comment by bybai** [comment]
fixed in #3201
